src/base/memory_monitor_sr.cpp

```diff
@@ -73,7 +73,7 @@ namespace cubmem
   {
     std::string filecopy (file);
 #if defined(WINDOWS)
-    std::string target ("\\src\\");
+    std::string target (""); // not supported
```

Comment:
```suggestion
    std::string target (""); // not supported
    assert_release (false);
```
To prevent an undefined behavior. Or, just assert(false); if it's not allowed asset_release() for this feature(MMON).

Reply:
I add assert(false). Maybe this function will not be called because I am not going to create `mmon_Gl` if cub_server is started in Windows despite `enable_memory_monitoring=yes`.